go.mod

```diff
@@ -50,8 +50,8 @@ require (
 	github.com/onflow/cadence v0.42.10
 	github.com/onflow/crypto v0.25.1
 	github.com/onflow/flow v0.3.4
-	github.com/onflow/flow-core-contracts/lib/go/contracts v0.15.1
-	github.com/onflow/flow-core-contracts/lib/go/templates v0.15.1
+	github.com/onflow/flow-core-contracts/lib/go/contracts v0.15.2-0.20240318135301-f59efe400a8b
+	github.com/onflow/flow-core-contracts/lib/go/templates v0.15.2-0.20240318135301-f59efe400a8b
```

Comment:
Have these changes to the core contracts on master already been ported to the stable cadence feature branch (in the core contracts repo)?

Reply:
There is a stable Cadence version (https://github.com/onflow/flow-core-contracts/pull/419), but this is pointing to the pre-stable Cadence version. Both are still on feature branches. 

Thanks for pointing this out. I was hoping to implement https://github.com/onflow/flow-go/issues/5828 (allows simplifying the pending smart contract change) before merging the pinned branches. But we should pin a merged and tagged version of `core-contracts` before merging to `master`. 

We're not planning to deploy the smart contract change any time soon (hence why it hasn't been merged) and there is only one integration test which depends on the smart contract change. I might just disable that test prior to merging, and go back to the same `core-contracts` version from `master`.

Reply:
Disabled the test and reverted to `v0.15.1` in https://github.com/onflow/flow-go/pull/5840/commits/06978a492d9fc7b4e67087120af0979e3396b894.